packages/design-system/src/components/Button/Button.stories.tsx

```diff
@@ -2,102 +2,90 @@ import React from 'react';
 import Button from './Button';
 import { NextIcon } from '../Icons';
 import Spinner from '../Spinner/Spinner';
+import type { Meta, StoryObj } from '@storybook/react';
 
-export default {
+const meta: Meta<typeof Button> = {
   title: 'Components/Button',
   component: Button,
-  argTypes: {
-    children: {
-      control: { type: 'text' },
-      type: { name: 'string', required: true },
-    },
-    disabled: {
-      control: { type: 'boolean' },
-    },
-    component: {
-      table: {
-        disable: true,
-      },
-    },
-    size: {
-      options: ['small', 'big'],
-    },
-    variation: {
-      options: ['solid', 'ghost'],
-    },
-  },
   args: {
     children: 'Your button text here',
     onDark: false,
   },
 };
+export default meta;
 
-const Template = ({ ...args }) => <Button {...args} />;
+type Story = StoryObj<typeof Button>;
 
-export const Default = Template.bind({});
-export const OnDark = Template.bind({});
-OnDark.args = {
-  onDark: true,
-};
-OnDark.parameters = {
-  // Must supply `layout: 'fullscreen'` when we use `onDark: true`
-  onDark: true,
-  layout: 'fullscreen',
-};
+export const Default: Story = {};
```

Comment:
Oh whoa, so we don't even need a render function when it's just the plain component?

Reply:
Yeah in cases where there are no args to pass in it's not needed, note to you and @zarahzachz if you don't have a render function with args the args will not show up as editable in SB. So if you run into a case where you aren't seeing args being configurable it's likely a missing render function with args passed..

```
render: function Component(args) { },
```